Library/Formula/mplayer.rb

```diff
@@ -50,19 +39,14 @@ def install
     # It turns out that ENV.O1 fixes link errors with llvm.
     ENV.O1 if ENV.compiler == :llvm
 
-    # we disable cdparanoia because homebrew's version is hacked to work on OS X
-    # and mplayer doesn't expect the hacks we apply. So it chokes.
```

Comment:
well, this formula's previous design is rather old - is cdparanoia still "hacked" to work on OS X? Also, it's not a requirement and if it's not a requirement, isn't it also not in the build path and therefore irrelevant? I can try playing a cd maybe late evening (which will be in 8 hours roughly)